deploy/chart/templates/0000_50_olm_08-catalog-operator.deployment.yaml

```diff
@@ -76,6 +87,8 @@ spec:
           - --client-ca
           - /profile-collector-cert/tls.crt
           {{- end }}
+          - --workload-user-id
```

Comment:
```suggestion
```
The idea of passing the USER ID will not work.
We have no way to know a fixed value number in OCP.

Reply:
The idea here for the downstream is to omit the flag and let SCC inject the user id in the range it wants.

Reply:
+1 to what Per said.

Reply:
> The idea here for the downstream is to omit the flag and let SCC inject the user id in the range it wants\

I understood the idea and that will not work.
We have no way to know the value to inform. What value will you pass when it will run in OCP? That is not a field value or a fixed range at all. 

Also, the good practices for Docker are to have the userID set in the image ( which sort out the issue ), remove all these complexities, and make it eaiser for us to keep the project maintained.